SR2020/2020-03-06-covid-19-website.md

```diff
@@ -0,0 +1,16 @@
+---
+to: sr2020-teams
+subject: COVID-19 Coronavirus Student Robotics Updates
+---
+
+# COVID-19 Coronavirus Student Robotics Updates
+
+Hello again,
+
+Following on from an email sent yesterday, we have now put a page on our website expanding further on how we're assessing the situation regarding COVID-19.
+
+You can find it here: https://studentrobotics.org/news/2020-03-11-sr2020-covid-19-updates/
```

Comment:
Should we link to the page that updates rather than the blog post announcing the page that updates?
```suggestion
You can find it here: https://studentrobotics.org/covid-19/
```